### ISSUE TITLE
Default case not working: size mismatch in channel_express function

### ISSUE THREAD

**Issue by seppia978** [RUIYUN-ML/ERM-KTP#1]
Hello, thank you for this implementation of your work!

I'm just running the standard default case, by using all the default input args, modifying only the variable `ifmask=True` during training (to test the ERM path only). 
It raised a RuntimeError at line 54, in "ERM_layer.py": `The size of tensor a (64) must match the size of tensor b (16384) at non-singleton dimension 1`.

Apparently `mask` and `x` have incompatible shapes `([256, 16384, 1, 1]` and `[256, 64, 8, 8]`, respectively).
How should the correct mask shape be? I was expecting the mask's shape to be `[256, 64, 1, 1]` (a scalar for each channel, for each image in the batch).

Thank you!

**Comment by Travaill** [comment]
Sorry for the late response. We have tested the code and we found it was a small mistake. The `channel_express` function is used in the KTP process to mask the channel of the target classes. Hence, you can change the line 164 in train_ERM.py to `outputs, regulization_loss = model(inputs, labels=labels)` and this issue will be sovled.
